templates/Layout/Page_results_solr.ss

```diff
@@ -2,11 +2,11 @@
     <h1>$Title</h1>
 
     <% if $Query %>
-        <p class="searchQuery">You searched for &quot;{$Query}&quot;</p>
+        <p class="searchQuery"><%t Results.SearchQuery 'You searched for' %> &quot;{$Query}&quot;</p>
```

Comment:
`Results.SearchQuery` is a global translation key, so it's not very unique. Perhaps you could use `SolrResultsPage.SearchQuery` instead?

Reply:
Thanks, I changed the key to be more unique.